frontend/src/components/auth/AuthTemplate.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from "react";
+import { Redirect } from "react-router-dom";
+import { Image, HStack, Text, VStack } from "@chakra-ui/react";
+
+import AuthContext from "../../contexts/AuthContext";
+import ImageType from "../../types/ImageTypes";
+import { HOME_PAGE } from "../../constants/Routes";
+import { JUMP_MATH_LOGO } from "../../assets/images";
+
+interface AuthTemplateInputProps {
+  title: string;
+  image: ImageType;
+  form: React.ReactElement;
+}
+
+const AuthTemplate = ({
```

Comment:
I also think that the subtitle and errors should be refactored into this general component...but this PR is getting kind of lengthy, so I'll open a subsequent one to address that!